setup.py

```diff
@@ -9,9 +9,9 @@
 install_requires = [
     'click==6.6',
     'botocore>=1.5.40,<2.0.0',
-    'virtualenv>=15.0.0,<16.0.0',
     'typing==3.5.3.0',
     'six>=1.10.0,<2.0.0',
+    'pip>=9'
```

Comment:
This is a bit tricky to do with pip I think, because while we're not setuptools style, we do rev our versions fairly regularly, but our breaking changes tend to be removing things that few people use rather than completely changing the common paths. So while we could do this, we should expect to need to rev this once or twice a year.